trento/xml/systemd-install.xml

```diff
@@ -681,6 +675,12 @@ firewall-cmd --reload
 systemctl enable --now nginx
 </programlisting>
       </step>
+             <step>
+          <para>
+            Copy the generated <literal>trento.crt</literal> to a location accessible by NGINX:
```

Comment:
This should be after the copy trento key step right ?

Reply:
Same as above. The new location for the crt file only exists once nginx has been installed.

Reply:
Yeah you were right :)